### PR TITLE
Check if regex and regex.test is available before calling it

### LINKED ISSUE #607: regex.test is not a function
@kadira/storybook v2.29.3

=> Loading custom .babelrc
=> Loading custom webpack config.
/Users/kitze/webdev/personal/x/client/node_modules/@kadira/storybook/dist/server/config.js:102
      return regex.test('my_package.json');
                   ^

TypeError: regex.test is not a function

Got this error with the newest version of storybook. Am I doing something wrong?

### PR DESCRIPTION
**Pull request by kitze**
Trying to fix #607

### CODE REVIEW

**Comment by arunoda** [comment]
We need to do this.
But I wanna know about the real cause of the #607 .
Could you reply to my comment on #607?

**Comment by kitze** [comment]
@arunoda just replied.

File: src/server/config.js [hunk 1]
```diff
@@ -11,7 +11,17 @@ const logger = console;
 export function addJsonLoaderIfNotAvailable(config) {
   const jsonLoaderExists = config.module.loaders.reduce(
     (value, loader) => {
-      return value || [].concat(loader.test).some(regex => regex && regex.test && regex.test('my_package.json'));
+      return value || [].concat(loader.test).some((loader) => {
```
Comment (arunoda) [inline src/server/config.js]: Could you rename this loader name into something like `matcher`.
Like this:

```js
return value || [].concat(loader.test).some((matcher) => { ...
```

**Review by arunoda — CHANGES_REQUESTED** [review]
After that, we are good to go.

File: src/server/config.js (hunk 1 above)
Comment (kitze) [inline src/server/config.js]: @arunoda done!

**Review by arunoda — APPROVED** [review]

**Comment by arunoda** [comment]
Awesome.

**Pull request merged**